web_src/js/index.js

```diff
@@ -242,6 +242,8 @@ function initReactionSelector(parent) {
         let react = content.find('.segment.reactions');
         if (!resp.empty && react.length > 0) {
           react.remove();
+        } else if (resp.empty && resp.html === '') {
+          react.remove();
```

Comment:
Shouldn't we still ensure that `react.length` is non-zero here?

Reply:
When click and there has no emoji left. The ajax XHR looks like this:
![image](https://user-images.githubusercontent.com/19653204/82346960-acb37480-9a29-11ea-801e-54837e74639d.png)

Reply:
This XHR response is unique. Don't need to do the superfluous check.

Reply:
`react.length` has nothing to do with `resp` however? You don't want to do `.remove` on something that doesn't exist.

Reply:
"react" is the jquery selector. "resp" is the XMLHTTPRequest's attribute.
Build a new version with this js and have a try.

Reply:
What I am concerned about is executing `.remove` on `react` if user clicks two times.